modules/logging-loki-logcli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+:_content-type: REFERENCE
+[id="logging-loki-logcli-about_{context}"]
+
+= Querying Loki
+
+You can use Loki's command-line interface `logcli` to query logs.
+
+.Example Application Log Query
+[source,terminal]
+----
+$ oc extract cm/lokistack-sample-ca-bundle --to=lokistack --confirm
+$ cat lokistack/*.crt >lokistack_ca.crt
+$ logcli -o raw --bearer-token="${bearer_token}" --ca-cert="lokistack_ca.crt" --addr xxxxxx
```

Comment:
This looks as if this file is not supposed to be in the PR because it contains almost no content and is not referenced in the other docs.